index.js

```diff
@@ -434,20 +441,27 @@ function main (options) {
           stack.push(value)
           let i = 0
           for (; i < value.length - 1; i++) {
-            const tmp = stringifySimple(i, value[i], stack)
+            const tmp = stringifySimple(i, value[i], stack, depth)
             res += tmp !== undefined ? tmp : 'null'
             res += ','
           }
-          const tmp = stringifySimple(i, value[i], stack)
+          const tmp = stringifySimple(i, value[i], stack, depth)
           res += tmp !== undefined ? tmp : 'null'
           stack.pop()
           return `[${res}]`
         }
 
         let keys = Object.keys(value)
+        if (options.maxBreadth) {
+          keys = keys.slice(0, options.maxBreadth)
+        }
```

Comment:
This is silently going to remove entries. That's not ideal for debugging. If something like that is included, it should somehow indicate that things where removed. Maybe add a fake entry such as: `"[DEBUG]": "${n} keys not stringified"`.

Ideally, this is also moved below the `maxDepth` check to prevent work in case it's already the most inner depth.

Reply:
I took this approach from another [library](https://github.com/davidmarkclements/fast-safe-stringify/pull/51/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R78). 

The idea is exactly to remove entries at current level of depth. Since the default value is _disabled_ for both features, the user should be aware of that.

I think adding a fake entry is a good idea 👍